lib/ansible/module_utils/tacp.py

```diff
@@ -196,9 +199,21 @@ def is_valid_uuid(uuid_to_test, version=4):
 
 def api_response_to_dict(api_response):
     if type(api_response) != ApiException:
-        message_str = api_response.message
-    else:
-        message_str = str(api_response)
+        if type(api_response) == str:
```

Comment:
I would go with the canonical way to check for type equality, which is using `isinstance`